recipe/meta.yaml

```diff
@@ -108,16 +108,16 @@ outputs:
         # shared
         - test -f $PREFIX/lib/libfaiss.so                        # [linux]
         - test -f $PREFIX/lib/libfaiss.dylib                     # [osx]
-        # - if not exist %PREFIX%\\Library\\bin\\faiss.dll exit 1  # [win]
+        - if not exist %PREFIX%\Library\bin\faiss.dll exit 1     # [win]
 
         # absence of static libraries
         - test ! -f $PREFIX/lib/libfaiss.a                       # [not win]
-        # - if exist %PREFIX%\\Library\\lib\\faiss.lib exit 1      # [win]
+        - if exist %PREFIX%\Library\lib\faiss.lib exit 1         # [win]
 
         # headers
         {% for each_header in headers %}
         - test -f $PREFIX/include/faiss/{{ each_header }} || (echo "{{ each_header }} not found" && exit 1)  # [unix]
-        # - if not exist %LIBRARY_INC%\\faiss\\{{ "\\".join(each_header.split("/")) }} exit 1                  # [win]
+        - if not exist %LIBRARY_INC%\faiss\{{ "\".join(each_header.split("/")) }} exit 1                     # [win]
```

Comment:
```suggestion
        - if not exist %LIBRARY_INC%\faiss\{{ "\\".join(each_header.split("/")) }} exit 1                     # [win]
```

Reply:
I just removed them based on the fact that the failing CI had (resolving `%LIBRARY_BIN%`):
```
set PREFIX=D:\bld\faiss-split_1607434453074\_test_env
set SRC_DIR=D:\bld\faiss-split_1607434453074\test_tmp
(base) %SRC_DIR%>call "%SRC_DIR%\conda_test_env_vars.bat" 
(base) %SRC_DIR%>set "CONDA_SHLVL="   && 
(base) %SRC_DIR%>conda activate "%PREFIX%" 
(%PREFIX%) %SRC_DIR%>IF 0 NEQ 0 exit /B 1 
(%PREFIX%) %SRC_DIR%>call "%SRC_DIR%\run_test.bat" 
(%PREFIX%) %SRC_DIR%>if not exist %PREFIX%\Library\bin\\faiss.dll exit 1 
```

I.e. the double-backslash before `faiss.dll` is conflicting (possibly?) I have no other solution with finding the dll?

Reply:
Nevermind, I misunderstood

Reply:
I just meant the `"\\"` within in the Jinja part because that one is "Python" syntax, i.e., they'll be printed as single `\` when the Jinja template is rendered.
The others should be single `\` as you had it.

Reply:
That's what I meant by "I misunderstood"... ;-)

Reply:
Though I'm still surprised, because this was taken verbatim from [arrow-cpp](https://github.com/conda-forge/arrow-cpp-feedstock/blob/master/recipe/meta.yaml#L139)

Reply:
```yaml
        {{ "- if %s exist %%PREFIX%%\\Library\\bin\\arrow_cuda.dll exit 1" % (['', 'not'][cuda_enabled]) }}   # [win]
```
is a Jinja/Python string that gets rendered with single backspaces.

Reply:
Given that YAML has at least 4 ways to write strings and then we even put on Jinja templating with its own string handling on top is just weird overall.
I won't blame anyone who gets confused there; happens all the time ;)